src/search.py

```diff
@@ -70,10 +70,6 @@ def get_page_views_statistic() -> Dict[str, int]:
     return page_views
 
 
-def get_client():
```

Comment:
What happened here? :)

Reply:
it is obsolete one, we don't use it anywhere